src/Microsoft.Extensions.CodeGenerators.Mvc/Controller/ControllerGeneratorModel.cs

```diff
@@ -17,7 +17,10 @@ public class ControllerGeneratorModel : CommonCommandLineModel
         [Option(Name = "controllerName", ShortName = "name", Description = "Name of the controller")]
         public string ControllerName { get; set; }
 
-        [Option(Name = "restWithNoViews", ShortName = "api", Description = "When specified, generates a Controller with REST style API, noViews is assumed and any view related options are ignored")]
-        public string IsRestController { get; set; }
+        [Option(Name = "restWithNoViews", ShortName = "api", Description = "Specify this switch to generate a Controller with REST style API, noViews is assumed and any view related options are ignored")]
```

Comment:
Good question, when I was writing this first time in CTP3 or CTP4 time frame, I think there was no resource support in DNX and I never got to fix this part (I am not myself sure what's the support right now) , We probably need a bug to track this ...